roda-ui/roda-wui/src/main/java/org/roda/wui/client/common/slider/InfoSliderHelper.java

```diff
@@ -184,7 +201,21 @@ public static void updateInfoSliderPanel(BrowseAIPBundle bundle, SliderPanel inf
     }
 
     if (StringUtils.isNotBlank(aip.getInstanceId())) {
-      values.put(messages.itemInstanceId(), createAipInstanceIdHTML(bundle));
+      String distributedMode = ConfigurationManager.getStringWithDefault(
+        RodaConstants.DEFAULT_DISTRIBUTED_MODE_TYPE.name(), RodaConstants.DISTRIBUTED_MODE_TYPE_PROPERTY);
+      if (distributedMode.equals(RodaConstants.DistributedModeType.CENTRAL.name())) {
+        String instancename = bundle.getInstanceName();
+        Anchor anchor = new Anchor();
+        if (StringUtils.isNotBlank(instancename)) {
+          anchor.setText(instancename);
+        } else {
+          anchor.setText(aip.getInstanceId());
+        }
+        anchor.setHref(HistoryUtils.createHistoryHashLink(ShowDistributedInstance.RESOLVER, aip.getInstanceId()));
+        values.put(messages.distributedInstanceLabel(), anchor);
+      } else {
+        values.put(messages.itemInstanceId(), createAipInstanceIdHTML(bundle));
+      }
```

Comment:
You should put this in a common method to avoid repeating code

Reply:
Done